### ISSUE TITLE
Should W3C continue to stagger AB/TAG elections ?

### ISSUE THREAD

**Issue by michaelchampion** [w3c/w3process#688]
I see this point raised  by @chaals in https://github.com/w3c/w3process/issues/484#issuecomment-749373454 :

> staggering terms, reduces the ability of any voting system to support diversity. This is the case under any voting system, even approval (which generally works against diversity). The purported benefit of staggered terms is to ensure that continuity provides some ongoing measure of institutional memory, rather than finding that everyone is new. That is a separate question, and I will raise a separate issue to discuss it as a proposal.

Apologies if Chaals has opened this issue and the CG or AB closed it ...But the current December 2022 situation where there are 4 vacant AB seats, 3 whose terms end in six months, makes this question very timely.

Strawman proposal:  Revise https://www.w3.org/2021/Process-20211102/#ABParticipation to eliminate  "Terms are staggered so that each year, either five or six terms expire.” And add some language about how to handle people whose terms would have expired in 2023 and 2024. (I don't have a proposal for that language).

Discussion:  I agree with Chaals that staggering terms reduces the ability of STV to support outcomes that allow representation of diverse geographical, business, philosophical, etc. groupings in proportion to the size of the electorate aligned on one or more of those dimensions.  The original motivation for staggering terms, which I also recall as "preserve institutional memory" seems rather hypothetical given the tendency for incumbents to be re-elected, even under STV.   It will take some work to find language to handle those elected for 2-year terms in 2022 if this change is adopted before the next AB election, or how to phase this in over multiple election cycles.

**Comment by cwilso** [comment]
I'm in strong agreement that we should stop staggering terms.

**Comment by igarashi50** [comment]
I agree on stopping staggering terms. I do not see the benefit of the "preserve institutional memory" for AB/TAG since of the tendency incumbents to be re-elected. Also, it is better for W3C to choose talented and diverse persons by electing all seats of AB/TAG with the STV voting.

**Comment by chaals** [comment]
I agree (with my older opinion) that staggered terms means the elections produce less representative results each time.

I am concerned about institutional memory, despite the tendency for incumbents to win re-election. I have repeatedly watched groups (outside the W3C as well) go over the same ground after a few years, and those who are able to look carefully at what happened last time seem to cover it far more effectively. That includes how they understand whether there is "something different this time". However, I think there are better mechanisms available than ensuring some people get to hang around when the membership have hanged their mind about what they are looking for.

As an ex-member of the AB, and a member of W3C, I'm perfectly able to provide my input - whether asked for it or not. And like others in that position, I do so. Groups can, and sometimes does, ask specific people outside the group for input, if they feel it would be helpful.

I would expect an all-new AB (or TAG or BoD - something I recall happening only once in the history of W3C) to do so relatively frequently. It is also the case that various individuals can and do ask those who were there before about things that have happened.

**Comment by chaals** [comment]
In terms of implementation, we already have a well-tried mechanism to manage elections where there is a shorter term for one or more candidates, and we're using it again in the current AB election. 

So we can remove the statement that terms are staggered, and make elections every two years - perhaps even years for TAG and BoD, odd years for AB. 

I don't like extending term lengths, especially not beyond the normal two years. But I do think it makes sense to adjust the timing of elections so they are evenly spaced.

As required by currently expiring terms we can hold special elections to fill vacancies.

For a concrete example, assuming we adopt a new process in late 2023*:

We would have an AB election in July 2024, for one-year terms of the seats that expire then. We then have an election of all AB seats in July 2025, 2027, and every other odd year.

We could set the TAG election of late 2023 to create short terms, with the assumption that in many cases this will give a big advantage to the incumbents, have an all-seats election in March 2024.

The complicated details (as a strawman implementation proposal): For that March 2024 election, members already elected until the end of 2024 would be eligible to run, and if elected their existing term would be replaced and their extra seat abolished. If they were not elected, they would serve their term in a slightly enlarged TAG, with their seat abolished at the end of that term to return the TAG to its current size. Under this proposal, if all incumbents from the current TAG election are re-elected in March 2024, the TAG would be exactly the same size. If none are, the TAG would have 3 extra members from April 2024 until the end of 2024.

Alternative approaches to the timing shift:
- run the 2023 election for terms of about 28 months, meaning we maintain staggered elections until 2026, or 
- declare in the new Process that the terms of all TAG members end in March 2024, avoiding the possibility of an extra-large TAG.

We would also ask the BoD to change the bylaws, so they have an election in 2023 for member-elected seats that expire, with a term running until October 2024, and in October 2024 they begin a cycle of elections every 2 years with all member-elected seats open for a 2-year term. (They also have to work out how to handle any timing gaps, but it's a case of picking one of the options outlined already).

*With a change to LE, I don't think this is the highest priority facing W3C. Getting deep agreement on how to move to a formally post-TimBL W3C seems at least as important as getting formal documentation in place quickly, and if we don't reach agreement on how to unstagger elections within the next 10 months we can always take the same options and shift them out by a year, or however long it takes.

**Comment by chrisn** [comment]
This discussion came up in the context of the current AB election. To clarify, is this issue about changing the rules for the current AB election (where the voting form makes it clear about the term lengths) or for future elections? My own view is that we shouldn't be changing the rules for an election that's in progress.

**Comment by chaals** [comment]
> This discussion came up in the context of the current AB election. To clarify, is this issue about changing the rules for the current AB election...?

I believe this is about changing the rules for future elections. The question has been raised before (e.g. I noted it [two years ago](https://github.com/w3c/w3process/issues/484#issuecomment-749373454)), but it seems @michaelchampion found it resonated in the circumstances of the current election.

I agree that changing rules for an election in progress is bad governance practice and should only happen for seriously urgent problems - and this isn't one.

I nominated for the AB election knowing that 3/4 of the current seats were 6-month terms, because it says so in the [call for nominations](https://lists.w3.org/Archives/Member/w3c-ac-members/2022OctDec/0015.html):

> This is a Call for Nominations for an election to begin on 2 December 
and end on 14 January to fill those vacancies. Three of the terms will 
end on 30 June 2023, one of the terms will end on 30 June 2024.

I also expect to vote based on that information.

**Comment by mprorock** [comment]
> > This discussion came up in the context of the current AB election. To clarify, is this issue about changing the rules for the current AB election...?
> 
> I believe this is about changing the rules for future elections. The question has been raised before (e.g. I noted it [two years ago](https://github.com/w3c/w3process/issues/484#issuecomment-749373454)), but it seems @michaelchampion found it resonated in the circumstances of the current election.
> 
> I agree that changing rules for an election in progress is bad governance practice and should only happen for seriously urgent problems - and this isn't one.
> 
> I nominated for the AB election knowing that 3/4 of the current seats were 6-month terms, because it says so in the [call for nominations](https://lists.w3.org/Archives/Member/w3c-ac-members/2022OctDec/0015.html):
> 
> > This is a Call for Nominations for an election to begin on 2 December
> > and end on 14 January to fill those vacancies. Three of the terms will
> > end on 30 June 2023, one of the terms will end on 30 June 2024.
> 
> I also expect to vote based on that information.

Thank you @chaals - I think this is extremely well said.  The Term Length in the nominations was very clear, and changing horses mid-stream would be highly problematic.  

Considering an adjustment to term lengths once the election goes through would certainly be possible if the new AB sees enough desire in from the AC and can clearly identify the problems that would be addressed by an adjustment (without introduction of new issues).  

This current case is a bit odd in that it is a result of the BoD election and seating leaving folks unable to fill both roles for time reasons, and should not be seen as the norm.

**Comment by tobie** [comment]
Pulled out @chaals's (very valid) concerns about preserving institutional memory into a separate issue: https://github.com/w3c/w3process/issues/689.

**Comment by frivoal** [comment]
Leaving aside the question of changing an ongoing election, which I don't think is reasonable, I am also somewhat skeptical of the benefits of unstaggering elections, mainly on two accounts:

----

First, one observation: life circumstances evolve. If an election happens at a time that isn't quite right for a candidate to stand up for election, they can catch the next one. All other things being equal, more frequent opportunities are more inclusive, particularly of people in less stable situations. Example: as an invited expert, it is non trivial to secure funding for TAG or AB work, and if things don't line up just right at the right time, even someone who could win an election may fail to get funded, and therefore to stand. But they could sort it out next year, if there's an election next year. Another one: someone on parental leave might miss an election, but they could run next year, if there's an election next year. Someone who just lost their job in an economic downturn might not find themself able to run for election. But they could run next year, once they're back on their feet, if there's an election next year. And so on. Now, this has to be weighted against other factors, and I'm not advocating for holding elections every month to maximize opportunities. But reducing the frequency of elections would reduce windows of opportunities.

----
On a different consideration, it seems to me that unstaggered elections strengthen the effects of STV. I am unsurprised that people who like STV in the first place would also like this notion (e.g. @chaals or @igarashi50), but I am perplexed that some people who have long argued against it also seem to support this (e.g. @michaelchampion @cwilso @dwsinger ).

The way I understand it, STV is indeed effective at supporting some forms of diversity, but in a very specific way that seems to me to be very appropriate for something like a national parliament, and a lot less for the AB or TAG: voters in a parliamentary election typically have some candidates or parties that that represent them best, and they would like to maximize the chance of *their candidate* to win, understanding that it's fair that others feel the same way too, so some tool that balances the representation of each constituency is good. If everybody wants their reppresentatives, ensuring that large groups don't drown out smaller ones is a good thing.

STV helps ensure that when everybody votes for their own, the diversity of the electorate ends up translated fairly well into the diversity of the elected people (with the Transferable part of STV dealing with rounding and accounting for secondary preferences). Moreover, electing more seats at the same time lowers the threshold that any particular group needs to reach to get their representative a seat, and so smaller groups can get representation, even if others would overlook them.

But I think the key assumption here is that everybody wants _their_ representative, and that _matching_ the diversity of the electorate in the elected people is the goal we're striving for. I am unconvinced that this is true for the specific elections we are discussing here.

* I think we are routinely trying to (and succeeding at) make the diversity of the elected people *exceed* that of the electorate. Look at the gender ratios in the AB and TAG, or the proportions of nationalities, or the proportion of people with disabilities, or a number of other characteristics, and compare with that of the AC. I believe that the AB and TAG are more diverse than what we'd get if everybody was trying to elect someone like themselves. I think that's a good thing, that we are successfully electing groups more diverse that the electorate seems indicative to me that most voters agree. 
* Relatedly, I think many voters in our elections, all while liking some candidates more and some less, are not typically trying to get *their own group* to win, and typically don't even recognize any particular group as being their own, and are actively trying to get a diverse and complementary set of people elected. People are actively seeking out candidates that bring something new to the group. I don't think we'd be getting the results we are if we weren't.

To me, this means that:
* In our context, it is generally unnecessary to lower the threshold to election (by increasing the number of simultaneous seats) to enable minority groups to be elected, because minority constituencies aren't alone in trying to get their person elected. Others are actively seeking them out too, and people who don't fit in any particular box and don't have a natural and well established constituency can still get elected.
* In our context, more than the under-represented groups, those who would most strongly benefit from more-seats-at-a-time-and-lower-thresholds are comparatively uncooperative groups, who would have a hard time getting votes from others can can only count on their own. That, in turn may entice other constituencies to regroup and defend their base.

Therefore, all in all, I worry that switching to unstaggered STV would encourage a vote-for-your-own mentality, crystalize factions, and might end up reducing the overall diversity of the elected bodies to merely reflecting that of the electorate, rather than exceeding it, as I believe we do today.

**Comment by agreiner** [comment]
We have several different issues arising here and getting a little bit conflated. There is the issue of whether we have staggered terms. There is the (somewhat perpetual) issue of what voting system we use. There is the original issue of how short a term is permissible on the AB. I agree that we should set aside the issue of making a change in the current election cycle, since altering the conditions during the election seems unfair to candidates as well as voters, would also be extra work for the Team, and would probably delay the result. 

On the question of staggering, even that one term suggests multiple options, depending on whether the stagger is accomplished by starting members on different dates or assigning shorter terms to some. If we consider staggering to be the former, simply removing the stagger does not decrease the probability of an incomplete term needing to be filled, other things being equal. Unless elected to fill the remainder of someone else's term, each member still serves the same length of time and has generally the same likelihood of resigning at any time during their term in both cases. Staggering just means that, if there are many resignations at once, we will likely have some partial terms to fill that are longer and some that are shorter. A board with everyone elected at the same time for the same term could still have members leave six months from the end of a term, so removing the stagger doesn't solve that problem. 

There are many ways that we could prevent offering very short terms, including electing all members, even replacements for resignations, to the same length term and simply allowing the group to be larger than usual for a while. We could also allow the board to be smaller for a time (not great when we lose multiple members at once). We could choose a compromise and say that a vacant seat with a term less than a year should be filled with a term that lasts one year plus the remaining time in the term, and that a vacant seat with a term more than a year should be filled for the remainder of the term. The latter would still result in short terms, but none less than a full year, and it would give us a bigger board for a while, but not as long as if all terms were for two years. In the case of multiple resignations, all the replacement terms would end up the same length. That seems a reasonable approach, but I'm still open to others.

I am not opposed to moving away from staggered elections. In fact, a little research shows me that there is some good evidence that staggered boards are less effective, but we need to make sure that we are solving the issues we want to solve as well. I would prioritize changes to the election process below addressing items related to the launch of W3C Inc.

**Comment by npdoty** [comment]
Thanks @cwilso and Tzviya for making that difficult decision and for explaining it. 

It seems not at all ideal to have an election for a 6-month position. I was aware of that time frame and the potential drawbacks when I chose to run and I don't think it would be appropriate to change that during the election. It's not ideal for potential members of the Advisory Board because they could just be getting up to speed and then potentially be voted out again, and it's not ideal for the membership because they may be asked to vote again after a short time (with maybe not much new information) and because they could have another sudden change in the people that they're working with to provide advice to the organization.

But I also understand that this was an exceptional situation, where the particular circumstances of the legal entity transition and the Board of Directors led to a sudden shift and a large number of vacancies during part of a term. I don't anticipate that this will be a common occurrence. Still, it's worth considering whether special elections for a seat with a term less than X months should instead be for term length + X months.

Staggered elections in general do increase the number of times we ask (with limited success) for the AC to vote and provide input. But I also see significant benefits, as @frivoal noted, of the opportunity of an election each year making it more accessible to people who may not have the exact support at the particular timing of an election every other year. And it provides an opportunity for the membership to consider a potential change in direction each year.

In addition to staggering terms, we are also staggering the TAG and AB elections. (With this particular special election, they overlapped, but that's unusual.) Would holding those elections at the same time every year make it easier for the Advisory Committee to give elections to W3C governance positions the sufficient attention that it deserves?

**Comment by michaelchampion** [comment]
@cwilso wrote:
>  think the psychological calculus of first choices is subtly different, and I would have hopes that the lower threshold and feeling like a voter already had one representative of their "group" would help lower the bar for others.

Well, in a non-staggered election system, no group "already had one representative."  But I agree with the logic:  With a low threshold, voters with multiple  affiliations can feel confident that their primary affiliation will be represented so they can give high ranked votes to  someone they see as representative of another group they feel affiliated with.   

For example, let's say I think of myself as a "core web platform interoperability" person but care about other matters such as sustainability; and  there are a couple strong candidates with web platform credentials plus one campaigning for more attention to sustainability.  With a high threshold, I'l give all my top rankings to the "web platform" candidates to help ensure at least one wins.  With a lower threshold, I am more likely to trust the voting system to ensure my primary affiliation group gets represented and give high a ranking to the sustainability candidate.  

Also, I suspect low thresholds will attract lots of candidates with loose affiliations to multiple geographical, industry, and philosophical groupings.  That SHOULD facilitate STV doing its "magic" to aggregate lots of information from diverse perspectives to choose those with the strongest overall support. Sure, those who abuse the system by disciplined bloc voting can easily get one candidate over the low threshold, but probably not get more than their bloc's "fair share" of seats overall.

**Comment by wareid** [comment]
I too understood the short term when I put myself forward for the election, but understand how challenging it will be to have to do this all over again in only a few months. I'm not sure it's worth exploring how this is done if this is a particularly rare occurrence though. 

I am ambivalent on staggered terms, I believe there are strong arguments for both sides. Considering we replace 50%+ of the board every time, I think each election already has the potential to bring in a lot of new perspectives. If that number were lower, I'd be more inclined to agree we need to change. 

I would like to dispel the diversity argument a bit though, as someone who has been focused on the topic within W3C for the last few years. Our current AB (prior to resignations) was I believe far more diverse than the membership it represented. The slate of candidates running for the AB is also diverse, and again, more so than the makeup of the membership itself. If what we want is diversity in the AB, how terms are constructed will not change that until we also change the makeup of the membership. Another email thread in AC-forum pointed out we only have a handful of member orgs from Africa, South America, and south east Asia. Obviously member orgs in those regions are not the only path to increasing the diversity of W3C, many of the current member orgs have offices or employees located in those regions (mine included), we need orgs to nominate, support, and encourage more participation from people in those regions. This is something I'd like to continue working on.

**Comment by torgo** [comment]
From a TAG chair perspective: I understand the arguments on both sides. Personally I think the staggered terms things makes more sense from the continuity of care perspective. However I also kind of feel like we should stay the course on election process while we allow this new W3C organization to solidify and then re-visit so we are not changing too many parts of the engines will we continue our flight.

**Comment by michaelchampion** [comment]
Closing for lack support, and since https://github.com/w3c/w3process/issues/697 exists
